src/renderers/shaders/ShaderChunk/lights_lambert_vertex.glsl

```diff
@@ -20,10 +20,10 @@ vLightFront = vec3( 0.0 );
 
 	for ( int i = 0; i < NUM_POINT_LIGHTS; i ++ ) {
 
-		IncidentLight directLight = getPointDirectLight( pointLights[ i ], geometry );
+		IncidentLight directIrradiance = getPointDirectIrradiance( pointLights[ i ], geometry );
 
-		float dotNL = dot( geometry.normal, directLight.direction );
-		vec3 directLightColor_Diffuse = PI * directLight.color;
+		float dotNL = dot( geometry.normal, directIrradiance.direction );
```

Comment:
I can rename it to incidentLight.  I do not want to get dragged into a long discussion of this as this is a very minor point.